models/reporting/commissioning/person_level/fct_person_sus_ae_recent.sql

```
@@ -56,6 +64,9 @@ SELECT
     , zeroifnull(a.ae_t1_12mo) as ae_t1_12mo
     , zeroifnull(a.ae_respiratory_attendance_12mo) as ae_lower_respiratory_attendance_12mo -- 500k million attendance >= admission
     , zeroifnull(ea.ae_respiratory_admission_12mo) as ae_lower_respiratory_admission_12mo --  15k admission > attendance
+    , zeroifnull(asc.asc_nel_12mo) as asc_nel_12mo
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)